contracts/core/fee-calculators/PerformanceFeeCalculator.sol

```diff
@@ -279,11 +263,34 @@ contract PerformanceFeeCalculator is IFeeCalculator {
         ScaleValidations.validateMultipleOfBasisPoint(parameters.streamingFeePercentage);
 
         require(
-            parameters.highWatermarkResetFrequency >= parameters.profitFeeFrequency,
+            parameters.highWatermarkResetPeriod >= parameters.profitFeePeriod,
             "PerformanceFeeCalculator.validateFeeParameters: Fee collection frequency must exceed highWatermark reset."
         );
     }
 
+    /**
+     * Verifies caller is valid Set. Calculates and returns streaming and profit fee.
+     *
+     * @return  uint256     Streaming Fee
+     * @return  uint256     Profit Fee
+     */
+    function calculateFees()
```

Comment:
For more flexibility for any contracts that might inherit this as a base contract, would pass in the `setAddress` as an argument